### DISCUSSION TITLE
Steve Jobs about customer experience

### DISCUSSION THREAD

**Discussion by iparthenios** [sdy60/syllabus#145] [Show and tell]
Δείτε το!
Μια απλή κουβέντα, μια ολόκληρη φιλοσοφία...
Συμφωνείτε;

https://x.com/i/status/1800218630124413349

**Comment by epidrome** [comment]
Σίγουρα αυτός ήταν πολύ συνεπής και πέτυχε να γίνει το δικό του (στην 2η προσπάθεια του). Κρίμα που αυτό δεν ήταν καλό για όλους εμάς :))
